src/Microsoft.PowerShell.MarkdownRender/FencedCodeBlockRenderer.cs

```diff
@@ -16,25 +16,28 @@ internal class FencedCodeBlockRenderer : VT100ObjectRenderer<FencedCodeBlock>
     {
         protected override void Write(VT100Renderer renderer, FencedCodeBlock obj)
         {
-            foreach (var codeLine in obj.Lines.Lines)
+            if (obj?.Lines.Lines != null)
             {
-                if (!string.IsNullOrWhiteSpace(codeLine.ToString()))
+                foreach (var codeLine in obj.Lines.Lines)
```

Comment:
`string` instead of `var` would be more obvious here

Reply:
Fixed